build/package/atlascli_mac_notarize.sh

```diff
@@ -20,16 +20,16 @@ set -Eeou pipefail
 # This depends on binaries being generated in a goreleaser manner and gon being set up.
 # goreleaser should already take care of calling this script as a hook.
 
-if [[ -f "./dist/macos_darwin_amd64/bin/atlas" && -f "./dist/macos_darwin_arm64/bin/atlas" && ! -f "./dist/mongodb-atlas-cli_macos_signed.zip" ]]; then
+if [[ -f "./dist/macos_darwin_amd64_v1/bin/atlas" && -f "./dist/macos_darwin_arm64/bin/atlas" && ! -f "./dist/mongodb-atlas-cli_macos_signed.zip" ]]; then
   echo "notarizing macOs binaries"
-  zip -r ./dist/mongodb-atlas-cli_amd64_arm64_bin.zip ./dist/macos_darwin_amd64/bin/atlas ./dist/macos_darwin_arm64/bin/atlas # The Notarization Service takes an archive as input
+  zip -r ./dist/mongodb-atlas-cli_amd64_arm64_bin.zip ./dist/macos_darwin_amd64_v1/bin/atlas ./dist/macos_darwin_arm64/bin/atlas # The Notarization Service takes an archive as input
   ./darwin_amd64/macnotary \
       -f ./dist/mongodb-atlas-cli_amd64_arm64_bin.zip \
       -m notarizeAndSign -u https://dev.macos-notary.build.10gen.cc/api \
```

Comment:
since you are changing this, when running `./darwin_amd64/macnotary -h` it states the following `-m, --mode string            specify one of the mode of operations: sign|notarize|signandnotarize (default "sign")` and we are sending `-m notarizeAndSign`, can you double check if we should change it?

Reply:
I have asked to the notarization team. Yesterday I showed them the command that we were running and said that it was correct. Let's see what they say. Thanks for pointing this

Reply:
I am waiting for them. I will add an update here as soon as I know more. I have also asked if we can move to linux

Reply:
Updates

Question: 
> I see that the notarization service is saying specify one of the mode of operations: sign|notarize|signandnotarize (default “sign”) but we are using notarizeAndSign . Which one is the correct one?

Answer:
> Yeah. The service has multiple modes. The one that you are using does both: signing and notarizing. That is the most common one. But if you need you can use either use just sign or notarize modes.

Question:
> Can we use the service on a linux machine? I am wondering if there is still the limitation on linux or not.

Answer:
> Yes you can use it on linux by downloading the linux binary. The only limitation is that you wont be able to use --verify flag. Which I beleive you are not using anyway